Source/Core/VideoCommon/VideoConfig.h

```diff
@@ -102,9 +102,9 @@ struct VideoConfig final
 
   // Utility
   bool bDumpTextures;
-  bool bHiresTextures;
-  bool bConvertHiresTextures;
-  bool bCacheHiresTextures;
+  bool bCustomTextures;
+  bool bConvertCustomTextures;  // what does it do?
```

Comment:
It converts custom textures from the old naming format (used in 4.0) to the new naming format (used in 5.0). There's a sticky in the custom texture section of the forums with more information about the naming format change.